tests/unit/scripting.tcl

```diff
@@ -113,6 +113,11 @@ start_server {tags {"scripting"}} {
         run_script {return redis.call('get',KEYS[1])} 1 mykey
     } {myval}
 
+    test {EVAL - is Lua able to call Server API?} { 
```

Comment:
Ideally let's add one more test. Something like:
```
test {EVAL - Make sure global server can be overwritten by local variable } {
    run_script {local server = 'whatever'; return server.call('get',server)} 0 
}
```

Just to make sure we can shadow the server variable with a local one.

Reply:
Added.